package.json

```diff
@@ -32,7 +32,12 @@
   "types": "dist/index.d.ts",
   "dependencies": {
     "@gnosis.pm/gp-v2-contracts": "^1.1.2",
+    "@types/qs": "^6.9.7",
+    "@uniswap/sdk-core": "^3.0.1",
     "ajv": "^8.8.2",
-    "ethers": "^5.5.3"
+    "cross-fetch": "^3.1.5",
+    "ethers": "^5.5.3",
+    "loglevel": "^1.8.0",
+    "qs": "^6.10.3"
```

Comment:
also, a bit reluctant to add dependencies... if it helps having a WIP, you can leave this one

Reply:
Have reduced dependencies as much as possible, but left cross-fetch and loglevel as I considered them important for cross environment compatibility and also to handle debugging scenarios (loglevel here prevents cluttering the console with console log messages)